cfgov/v1/models/molecules.py

```diff
@@ -75,3 +77,41 @@ class TextIntroduction(blocks.StructBlock):
     class Meta:
         icon = 'title'
         template = 'v1/demo/molecules/text_introduction.html'
+
+
+class ContactAddress(blocks.StructBlock):
+    label = blocks.CharBlock(max_length=50)
+    title = blocks.CharBlock(max_length=100, required=False)
+    street = blocks.CharBlock(max_length=100)
+    city = blocks.CharBlock(max_length=50)
+    state = blocks.CharBlock(max_length=25)
+    zip_code = blocks.CharBlock(max_length=15, required=False)
+
+    class Meta:
+        template = 'wagtail/molecules/contact-address.html'
+        icon = 'mail'
+        label = 'Address'
+
+
+class ContactEmail(blocks.StructBlock):
+    emails = blocks.ListBlock(atoms.Hyperlink(label='Email'))
+
+    class Meta:
+        icon = 'mail'
+        template = 'wagtail/molecules/contact-email.html'
+        label = 'Email'
+
+
+class ContactPhone(blocks.StructBlock):
+    fax = blocks.BooleanBlock(default=False, required=False, label='Fax?')
```

Comment:
Or on the flip-side, should the label `Email` on [L:97](https://github.com/cfpb/cfgov-refresh/pull/1127/files#diff-b43ebc1f91b7bbbe1b98a168af29a970R97) have a question mark?